lib/cartopy/mpl/gridliner.py

```diff
@@ -130,37 +160,51 @@ def __init__(self, axes, crs, draw_labels=False, xlocator=None,
 
         #: The :class:`~matplotlib.ticker.Locator` to use for the x
         #: gridlines and labels.
-        self.xlocator = xlocator or degree_locator
+        if mlocator is not None:
+            if not isinstance(mlocator, mticker.Locator):
+                mlocator = mticker.FixedLocator(mlocator)
+                print('fixed lon')
```

Comment:
Remove.

Reply:
Ok. 
However, these lines originally in the geoaxes.py module: https://github.com/SciTools/cartopy/blob/696e6df5ca09079e51d22c8c8686ebf7441340b6/lib/cartopy/mpl/geoaxes.py#L1204
Maybe this piece of preprocessing can be directly integrated onto gridliner.py?

Reply:
I meant the `print`...